### ISSUE TITLE
Tu-95 series corrections (CWDB and DB3000)

### ISSUE THREAD

**Issue by SunlitZelkova** [PygmalionOfCyprus/cmo-db-requests#1485]
### DB Selector

Both

### Affected DBID(s)

Aircraft_200, Aircraft_199, Aircraft_1325, Aircraft_133, Aircraft_643, Aircraft_771, Aircraft_2459, Aircraft_2458, Aircraft_1611, Aircraft_62, Aircraft_447, Aircraft_773, Aircraft_520

### Summary of Changes

There are a number of major inaccuracies with the Tu-95 series of aircraft, most notably a problem with the range.

For the Cold War DB:

Changes for **NK-12MV #1** engine-
- Military Static SFC at S/L should be changed to 0.21 kg/h/kg. It is currently too high and this is likely what is causing the range issue, as the fuel load is more or less accurate. If issues persist, it may be a problem with fuel consumption specifications, but there is no available data for this so the DB manager will need to edit it on their own to match the known range characteristics.

Changes for Aircraft_200 **Tu-95M Bear A**-
- Change the Strike radius for all combat loadouts (i.e. ones with weapons) to 7,020 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the YaD radar with the Short Horn [RPB-4 Rubin-1] radar. It should have identical detection and tracking arcs as Aircraft_677 Tu-16R.
- Replace the Box Tail [PRS-4 Krypton] gun director radar with the Bee Hind [PRS-1 Argon]. It should have identical detection and tracking arcs to the PRS-4 it currently has.
- Replace the SPS-3 jammer with the SPS-2 jammer. Arcs should remain identical.
- Replace the SPO-3 Sirena-3 RWR with the SPO-2 Sirena-2 RWR. Arcs should remain identical.
- Remove the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT.
- Change the fuel load to 84,440 kilograms.
- Change the 8x FAB-1500M-46 loadout to 10x FAB-1500M-46 bombs.

Changes for Aircraft_199 **Tu-95KM Bear C**-
- Change the Strike radius for the 1x AS-3 Kangaroo loadout to 4,455 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-5 Vishnya COMINT and SRS-4 Romb ELINT. It should have identical detection and tracking arcs to the ESM sensors it currently has.
- Change the fuel load to 84,440 kilograms.
- Change in-service date to 1962, and change the comment that comes after the name as well to "1962".

Changes for Aircraft_1325 **Tu-95KD Bear B**-
- Change the Strike radius for the 1x AS-3 Kangaroo loadout to 4,455 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-5 Vishnya COMINT and SRS-4 Romb ELINT. It should have identical detection and tracking arcs to the ESM sensors it currently has.
- Change the fuel load to 84,440 kilograms.
- Remove the 2x RN-28 Tactical Bomb loadout. The Tu-95KD had all bomb shackles removed.
- Remove the Bombsight - Ballistic Computing Property.

Changes for Aircraft_133 **Tu-95RT Bear D**-
- Change the Maritime Surveillance radius of that loadout to 7,600 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the Box Tail [PRS-4 Krypton] gun director radar with the Bee Hind [PRS-1 Argon]. It should have identical detection and tracking arcs to the PRS-4 it currently has.
- Replace the SPS-3 jammer with the SPS-2 jammer. Arcs should remain identical.
- Replace the SPO-3 Sirena-3 RWR with the SPO-2 Sirena-2 RWR. Arcs should remain identical.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-4A Romb-A ELINT, with identical arcs to the current SRS-7.
- Remove the Mushroom [RPB-1 Initsiativa] radar. This sensor does not exist, it is a result of Western observers misinterpreting what the missile guidance antenna for the Uspech-1A radar was.
- Remove one of the Generic Visual Reconnaissance Camera.
- Change the fuel load to 84,440 kilograms.
- Change aircraft name to Tu-95RTs. "Tu-95RT" is incorrect.

For the DB3000:

Changes for **NK-12MV #1** engine-
- Military Static SFC at S/L should be changed to 0.21 kg/h/kg. It is currently too high and this is likely what is causing the range issue, as the fuel load is more or less accurate. If issues persist, it may be a problem with fuel consumption specifications, but there is no available data for this so the DB manager will need to edit it on their own to match the correct range characteristics.

Changes for **NK-12MP #1** engine-
- Military Static SFC at S/L should be changed to 0.21 kg/h/kg. It is currently too high and this is likely what is causing the range issue, as the fuel load is more or less accurate. If issues persist, it may be a problem with fuel consumption specifications, but there is no available data for this so the DB manager will need to edit it on their own to match the correct range characteristics.

Changes to Aircraft_643 **Tu-95KM Bear C**-
- Change the Strike radius for the 1x AS-3 Kangaroo loadout to 4,455 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-5 Vishnya COMINT and SRS-4 Romb ELINT. It should have identical detection and tracking arcs to the ESM sensors it currently has.
- Change the fuel load to 84,440 kilograms.

Changes to Aircraft_771 and Aircraft_2459 **Tu-95K-22 Bear G**-
- Change the Strike radius for all combat loadouts (i.e. ones with weapons) to 4,455 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-5 Vishnya COMINT and SRS-4 Romb ELINT. It should have identical detection and tracking arcs to the ESM sensors it currently has.
- Change the fuel load to 84,440 kilograms.

Changes for Aircraft_133 and Aircraft_2458 **Tu-95RT Bear D**-
- Change the Maritime Surveillance radius of that loadout to 7,600 nautical miles.
- Change the Empty Weight to 84,300 kilograms.
- Replace the Box Tail [PRS-4 Krypton] gun director radar with the Bee Hind [PRS-1 Argon]. It should have identical detection and tracking arcs to the PRS-4 it currently has.
- Replace the SPS-3 jammer with the SPS-2 jammer. Arcs should remain identical.
- Replace the SPO-3 Sirena-3 RWR with the SPO-2 Sirena-2 RWR. Arcs should remain identical.
- Replace the SRS-6 Romb-4A COMINT and the SRS-7 Romb-4B ELINT with the SRS-4A Romb-A ELINT, with identical arcs to the current SRS-7.
- Remove the Mushroom [RPB-1 Initsiativa] radar. This sensor does not exist, it is a result of Western observers misinterpreting what the missile guidance antenna for the Uspech-1A radar was.
- Remove one of the Generic Visual Reconnaissance Camera.
- Change the fuel load to 84,440 kilograms.
- Change aircraft name to Tu-95RTs. "Tu-95RT" is incorrect.

Changes for Aircraft_1611 **Tu-95MS-6 Bear H**-
- Change the Strike radius for the 6x Kh-55SM loadout to 5,670 nautical miles.
- Change the Empty Weight to 91,200 kilograms.
- Change the Maximum Weight to 185,000 kilograms.
- Change the Max Payload to 7,800 kilograms.
- Change the fuel load to 87,000 kilograms.

Changes for Aircraft_62 and Aircraft_447 **Tu-95MS-16 Bear H**-
- Change the Strike radius for the 6x Kh-55SM loadout to 5,670 nautical miles.
- Change the Empty Weight to 91,200 kilograms.
- Change the Maximum Weight to 185,000 kilograms.
- Change the Max Payload to 20,800 kilograms.
- Change the fuel load to 87,000 kilograms.

Changes for Aircraft_773 and Aircraft_520 **Tu-95MSM Bear H**-
- Change the Strike radius for the 6x Kh-55SM loadout to 6,000 nautical miles. The Tu-95MSM utilizes new propellers and new engines for "increased range", but the exact increase is unknown, so I decided to simply round it up to an even 6,000, which I thought was not too extreme and not too conservative. The DB manager may change it as they choose however.
- Change the Strike radius for the 6x Kh-555 loadout to 6,000 nautical miles. The Tu-95MSM utilizes new propellers and new engines for "increased range", but the exact increase is unknown, so I decided to simply round it up to an even 6,000, which I thought was not too extreme and not too conservative. The DB manager may change it as they choose however.
- Change the Strike radius for the 8x Kh-101 loadout to 5,900 nautical miles. The Tu-95MSM utilizes new propellers and new engines for "increased range", but the exact increase is unknown, so I decided to simply round it up to an even 5,900, which I thought was not too extreme and not too conservative. The DB manager may change it as they choose however.
- Change the Strike radius for the 8x Kh-102 loadout to 5,900 nautical miles. The Tu-95MSM utilizes new propellers and new engines for "increased range", but the exact increase is unknown, so I decided to simply round it up to an even 5,900, which I thought was not too extreme and not too conservative. The DB manager may change it as they choose however.
- Change the Empty Weight to 91,200 kilograms.
- Change the Maximum Weight to 185,000 kilograms.
- Change the Max Payload to 20,800 kilograms.
- Change the fuel load to 87,000 kilograms.

Note on the Tu-95MSM changes- although the Tu-95MSM does use new engines not present in the DB, because the NK-12MVs in the DB are already capable of reaching the "new" range anyways, I didn't see a point in requesting a change as it would basically be cosmetic, considering the exact performance specifications aren't known anyways.

Note on the Tsar Bomba- technically, the "most correct" option would be to remove the Tsar Bomba loadout from the Tu-95M and create a new unit called Tu-95V, which would solely be equipped with Tsar Bomba. It could have the correct fuel statistics and proper range, while the change I proposed blocks unrealistic use in an AI mission but would leave it still capable of extreme, unrealistic range under direct player control.  I did not feel as though this would be worth it though, but if the DB manager thinks that would be be better, then I can provide the info for the Tu-95V.

### Sources

**Famous Russian Aircraft: Tupolev Tu-95 & Tu-142** by **Yefim Gordon** and **Dmitry Komissarov**, pages 39 (for bombload), 71 (for info about Tsar Bomba I used to make range estimate), 109 (for Tu-95KD data), 112, 113 (for Tu-95KM sensors and performance), 165, 166 (for Tu-95RTs sensors), 272 (for Tu-95MSM changes), 290 (for SFC of all engines), 295, 296 (for ECM), 347 (for performance characteristics)

**The Kremlin's Nuclear Sword: The Rise and Fall of Russia's Strategic Nuclear Forces, 1945-2000** by Steven J. Zaloga, page 26 (for Tu-95M range)

**Comment by Huib18** [comment]
@PygmalionOfCyprus , is it possible to lift this upgrade in priority? The Tu-95 plays a important role in the Ukraine conflict and in a lot of scenario's with NATO and even in Asia.
Together with a nice picture to turn it on ;-)
![FtbZtLFXsAEWM7F](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/60408110/7945fee7-2d43-48d2-93c4-2eab2c471420)

**Comment by PygmalionOfCyprus** [comment]
^ Just for the pretty picture. ;)

**Comment by PygmalionOfCyprus** [comment]
With deepest regrets this one is being pushed back to 503, we just had way too much on our plate this cycle to tackle a ticket of this size.

**Comment by Huib18** [comment]
Good luck then, and thanks for the hard work!

**Comment by claudejdev** [comment]
Hey there,

re. the  Aircraft_773 and Aircraft_520 Tu-95MSM Bear H :

according to : 
  -- Butowski, Piotr. Russia's Warplanes: Russian-Made Aircraft and Helicopters Today. Houston, Harpia Publishing, 2016.
  -- Jane's All the World's Aircraft In Service 2021-2022. Coulsdon, Surrey, UK: IHS Global.
  -- [[Actu] Modernisation du Tupolev Tu-95MS – Red Samovar](https://redsamovar.com/2021/10/22/__trashed/)

The Comments field should be deleted, as this platform is not to be confused with the "Tu-95MS-16", which is the "62 and Aircraft_447 Tu-95MS-16 Bear H".

As the upgrade contract was only signed in 2009 and the first one was delivered in December 2014. Id. 773's Year could be changed to _2014_, at the earliest.

That upgrade, referred to as M1 by Butwoski, did bring an A7371 satellite navigation receiver. A 233 "Generic Satellite Downlink"

A second stage of the upgrade, more ambitious, was performed subsequently, and the first delivery occurred in August 2020.
Thus, I'd suggest id 520's year be set at 2020.
  
That second stage of the upgrade notably includes 5 LCD screens and the replacement of the Obzor radar, with NV1.021 Novella radar.
Accordingly, I suggest changing the cockpit gen to _Glass Cockpit (Partial)_ and replacing the Obzor-MS radar with the _6589 NV-70M Novella_, as a placeholder.
Also, it removed the tail gunner installation. Its FC radar and weapons could then be removed.

 RedSamovar mentions that the NK-12MPM doesn't change its rating, as its main improvement is a longer service life. Butowski mentions that the heavier propeller improves the cruise efficiency by 0.01, and significantly reduces the vibration.

**Comment by PygmalionOfCyprus** [comment]
Handled 503.

The ranges confuse me, though. A cursory glance at Wikipedia is showing the Tu-95MS ferry range as 8100nmi. Sure, adjust a little up/down for upgrades, apply the ol' Wikipedia fuzz, and you might be a bit higher or lower, but nowhere near the ~15k nmi that a ~7000nmi combat radius would require. Am I missing something?

**Comment by claudejdev** [comment]
**Николай Якубович - 'Стратосферные крепости'' Б-52, М-4 и Ту-95 (2011, ВЭРО Пресс, Яуза, Эксмо)**
_Nikolai Yakubovich - 'Stratospheric Fortresses' B-52, M-4 and Tu-95 (2011, VERO Press, Yauza, Eksmo)_

![Война и мы  Авиаколлекция  Николай Якубович - 'Стратосферные крепости'' Б-52, М-4 и Ту-95 (2011, ВЭРО Пресс, Яуза, Эксмо)](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/d0bfc2ce-565d-4a66-b03d-ecdaf9be259e)

_The table shows the basic characteristics of the Hk-12 family of diesel engines._

Engine | Power T/O hp | Power cruise hp | SFC T/O kg-hp-h | SFC cruise | Compression ratio | Rated life hours | Weight kg 
---|---|---|---|---|---|---|---
Hk-12 | 12500 | 6500 | 0.225 | 0.164 | 9.5 | 100 | 2900 
Hk-12M | 15000 | 6500 | - | 0.158 | 9.5 | 300 | 2900 
Hk-12MV | 15000 | 6500 | 0.21 | 0.158 | 9.3 | 5000 | 3065 
Hk-12MP | 15000 | 6500 | 0.205 | 0.161 | - | - | -

> Note: 1) Altitude 11000 m at number M=0.68; 2) The calendar life of the Hk-12MV and Hk-12MP engines in 2008 was 2.3 years.

![Война и мы  Авиаколлекция  Николай Якубович - 'Стратосферные крепости'' Б-52, М-4 и Ту-95 (2011, ВЭРО Пресс, Яуза, Эксмо)](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/633aa8eb-a655-4ccb-b2c0-56d050f35a93)

So, for the MS :
- max range would be 14 100 km (cf. note 8) aka 7613 nm;
- "practical" range would be 11 600 km aka 6263 nm;
- with a 9 ton payload : 10 500 km aka 5670 nm;
- with a 20 ton payload : 6 500 km aka 3510 nm.

If I can read that correctly. 

Jane's seems to be off, while Butowski seems to match with that Russian literature.

Well, enjoy.

**Comment by SunlitZelkova** [comment]
> Handled 503.
> 
> The ranges confuse me, though. A cursory glance at Wikipedia is showing the Tu-95MS ferry range as 8100nmi. Sure, adjust a little up/down for upgrades, apply the ol' Wikipedia fuzz, and you might be a bit higher or lower, but nowhere near the ~15k nmi that a ~7000nmi combat radius would require. Am I missing something?

Hi,

All of the cruise missile carrying Tu-95s have a significantly lower combat range. I was unable to find the exact range in the Yefim Gordon book for the bomb carrying Tu-95 (Tu-95M) and so had to go with the vague number given in Zaloga's book. Note, however, that the ferry range of the Tu-95M as given by Yefim Gordon is on the order of 14,000 nautical miles.

Looking at the range from the Arctic and Siberian forward deployment bases the Tu-95s were supposed to take off from in a nuclear war, a 6263 nautical mile range given by claudejdev would not impede the ability of the aircraft to carry out the missions they were historically supposed to in the same way the pre-db503 ranges did. If the 7100 range makes these aircraft overpowered or impacts their flight performance, it could be changed as needed. If not, I don't think it is worth occupying the busy DB manager's time.

**Comment by Huib18** [comment]
Goodmorning Sir,

Congrats on 1328.13 and 504, yet another great update!

I see on the forum more calls about more depth/dispersion in the radar
bands. Getting this done unclass can be challenging and a very
time consuming business.
I came across this location: https://www.radars.org.uk/ which offers a
Radar Electronic Data Base (REDB).
This man was in the EW business for years and has compiled a lot of unclass
radar data. It is not for free but can give CMO a very big jump start.
Quote: The REDB now contains over 35,400 modes of parametric data,
detailing more than 16,500 different types of radar and related emitter
systems, deployed globally.
His master index is here:
https://secure.toolkitfiles.co.uk/clients/16529/sitedata/files/REDB-Index-231204.xlsx

Also a lot of Chinese stuff which can be helpful with your China update
process.
I found it better to mail this in person than to share this professional
stuff for the whole world ;-).

Maybe it is a step too far for CMO but it is better to have some options in
your pocket.

Thanks for your time and effort,

Huib18





Op vr 1 sep 2023 om 17:58 schreef Ethan Hermanson ***@***.***
